spec/fixtures/pulpcore-migration/katello-answers-after.yaml

```diff
@@ -21,6 +20,7 @@ foreman::plugin::dlm: false
 foreman::plugin::dhcp_browser: false
 foreman::plugin::expire_hosts: false
 foreman::plugin::git_templates: false
+foreman::plugin::google: false
```

Comment:
This is a modification that's missing in the other fixtures, causing the tests to fail.